src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/BreadcrumbContext.ts

```diff
@@ -0,0 +1,23 @@
+import { createContext, Dispatch, SetStateAction } from 'react';
+
+interface IBreadcrumbContextProps {
+    name: string;
+    setName: Dispatch<SetStateAction<string>>;
+    patientId: string;
+    setPatientId: Dispatch<SetStateAction<string>>;
+    schemaNumber: string;
+    setSchemanumber: Dispatch<SetStateAction<string>>;
+}
+export const BreadcrumbContext = createContext<IBreadcrumbContextProps>({
+    name: '',
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    setName: () => {},
+    patientId: '',
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    setPatientId: () => {},
+    schemaNumber: '',
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    setSchemanumber: () => {},
```

Comment:
I think it's possible to do something like this

```suggestion
/* @typescript-eslint:disable: no-empty-function */
export const BreadcrumbContext = createContext<IBreadcrumbContextProps>({
    name: '',
    setName: () => {},
    patientId: '',
    setPatientId: () => {},
    schemaNumber: '',
    setSchemanumber: () => {},
```

Reply:
I added a rule in the eslint config that fixed this issue.